### PR TITLE
Update install_ossim.sh

### PR DESCRIPTION
**Pull request by kalxas**
Moved a direct commit to another branch and opened a PR to discuss.

### CODE REVIEW

File: bin/install_ossim.sh [hunk 1]
```diff
@@ -7,27 +7,17 @@
 # Licensed under the GNU LGPL version >= 2.1.
 #
 
-if [ "$#" -lt 1 ] || [ "$#" -gt 1 ]; then
```
Comment (kalxas) [inline bin/install_ossim.sh]: @epifanio the script must stop immediately if no arguments are passed, so this should not be removed from here...

File: bin/install_ossim.sh [hunk 2]
```diff
@@ -7,27 +7,17 @@
...
 ./diskspace_probe.sh "`basename $0`" begin
 BUILD_DIR=`pwd`
 ####
 
+
 if [ -z "$USER_NAME" ] ; then
    USER_NAME="user"
 fi
 
+#USER_NAME="user"
```
Comment (kalxas) [inline bin/install_ossim.sh]: This is not needed, the above lines cover this

File: bin/install_ossim.sh [hunk 3]
```diff
@@ -35,27 +25,62 @@ APP_DATA_DIR="$BUILD_DIR/../app-data/ossim"
 DATA_FOLDER="/usr/local/share/data"
 OSSIM_VERSION=1.8.19
 BUILD_DATE=20150707
-mkdir -p "$TMP_DIR"
 
 apt-get -q update
 
-apt-get install --assume-yes ossim-core libossim-dev
+"""
```
Comment (kalxas) [inline bin/install_ossim.sh]: @epifanio is that supposed to be a comment here?

File: bin/install_ossim.sh [hunk 4]
```diff
@@ -35,27 +25,62 @@ APP_DATA_DIR="$BUILD_DIR/../app-data/ossim"
...
+                             libopenthreads14 libc6 libgcc1 libstdc++6 libgdal1h libgeos-c1 libgeos-3.4.2 libqt4-qt3support
+
+"""
+
+
 
-# apt-get install --assume-yes libtiff5 libfreetype6 libcurl3 libexpat1 libpng3 libfftw3-3  \
-#                              libgeotiff2 libqt4-core libqt4-opengl libpodofo0.9.0 libopenscenegraph99 \
-#                              libopenthreads14 libc6 libgcc1 libstdc++6 libgdal1h libgeos-c1 libgeos-3.4.2 libqt4-qt3support
+apt-get install --assume-yes libfftw3-bin libfftw3-long3 libfftw3-quad3 libgtkglext1 libilmbase6 \
```
Comment (kalxas) [inline bin/install_ossim.sh]: @epifanio Are those ossim-qt dependencies? All of them?

**Comment by kalxas** [comment]
@epifanio  I have already worked on this installer here: 
https://github.com/kalxas/OSGeoLive/commit/549b630c0059bebe38f90f8b6997d36b7165052a
can you please add your changes based on that commit?

**Comment by kalxas** [comment]
The build using the above commit throws this error:

--2015-08-12 20:54:34--  http://download.osgeo.org/livedvd/data/ossim/deb/ossim-qt_1.8.18_amd64.deb
Resolving download.osgeo.org (download.osgeo.org)... 140.211.15.132
Connecting to download.osgeo.org (download.osgeo.org)|140.211.15.132|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 5994108 (5.7M) [application/x-debian-package]
Saving to: 'ossim-qt_1.8.18_amd64.deb'

     0K ........ ........ ........ ........ ........ ........ 52% 1.35M 2s
  3072K ........ ........ ........ ........ ........ ...     100% 6.23M=2.7s

2015-08-12 20:54:37 (2.15 MB/s) - 'ossim-qt_1.8.18_amd64.deb' saved [5994108/5994108]

Selecting previously unselected package ossim-qt.
(Reading database ... 179251 files and directories currently installed.)
Preparing to unpack ossim-qt_1.8.18_amd64.deb ...
Unpacking ossim-qt (1.8.18) ...
dpkg: dependency problems prevent configuration of ossim-qt:
 ossim-qt depends on libopenscenegraph99; however:
  Package libopenscenegraph99 is not installed.

dpkg: error processing package ossim-qt (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 ossim-qt

**Comment by kalxas** [comment]
@epifanio I merged your changes with mine in this last commit. Can you please review?
Thanks.

**Comment by kalxas** [comment]
OK, more merges based on @epifanio comments

**Comment by kalxas** [comment]
@epifanio seems like there is an error in package ossim-plugins. See the build log:

--2015-08-13 11:31:33--  http://download.osgeo.org/livedvd/data/ossim/deb/ossim-plugins_1.8.18_amd64.deb
Resolving download.osgeo.org (download.osgeo.org)... 140.211.15.132
Connecting to download.osgeo.org (download.osgeo.org)|140.211.15.132|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1732362 (1.7M) [application/x-debian-package]
Saving to: 'ossim-plugins_1.8.18_amd64.deb'

     0K ........ ........ ........ ..                        100%  764K=2.2s

2015-08-13 11:31:35 (764 KB/s) - 'ossim-plugins_1.8.18_amd64.deb' saved [1732362/1732362]

Selecting previously unselected package ossim-plugins.
(Reading database ... 179763 files and directories currently installed.)
Preparing to unpack ossim-plugins_1.8.18_amd64.deb ...
Unpacking ossim-plugins (1.8.18) ...
dpkg: dependency problems prevent configuration of ossim-plugins:
 ossim-plugins depends on liblas-c2; however:
  Package liblas-c2 is not installed.

dpkg: error processing package ossim-plugins (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 ossim-plugins

**Comment by johanvdw** [comment]
Is there a source package for ossim? We could just add it to launchpad.

**Comment by kalxas** [comment]
For ossim-core, the source package is already updated in nightly ppa.
For the ossim-plugins and ossim-qt, I think fpm was used, so no source packages.

**Comment by johanvdw** [comment]
Please share the package, preferably on the debian git repository. Note that you can rebuild the package to use the latest version liblas which was uploaded to the nightly ppa. Now you will be linking to an older version, and two versions will be installed.

**Comment by kalxas** [comment]
@epifanio can you please point us to the sources used for building the packages?
( I assume it is this one, at least this is the source I used to build ossim-core deb: http://download.osgeo.org/ossim/source/latest/ossim-1.8.18-1.tar.gz )
Also, can you please provide some details on which plugins/apps are built?
Hopefully we will find the time to build proper debian packages...

**Comment by epifanio** [comment]
Yes I used the same source.

**Comment by kalxas** [comment]
In the latest build (150) the ossim applications seem to work fine when launched from desktop launchers, so I do not think we need a rebuild.
A weird issue is that the Geospatial menu launchers do not work.
It could be that -p argument in the launchers.....

**Comment by kalxas** [comment]
If there are no objections within 24h I will merge this PR.
Thanks.

**Pull request merged**